packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol

```diff
@@ -529,7 +528,7 @@ contract PricelessPositionManager is FeePayer {
 
         // Get the current settlement price and store it. If it is not resolved will revert.
         if (contractState != ContractState.ExpiredPriceReceived) {
-            expiryPrice = _getOraclePrice(expirationTimestamp);
```

Comment:
to accommodate the two kinds of oracle requests we can now request an `Expiration` oracle request or a `Liquidation` oracle request.